### ISSUE TITLE
TE Title not visible on widget in dark mode

### ISSUE THREAD

**Issue by kadrihvl** [toggl/mobileapp#7312]
## ⛑️ User Support

Running Time Entry Widget displays TE description in black (not visible) when app is in dark mode. 

## About the user

App version: Giskard/2.14
Phone OS: Android 9
Phone model: G8441

## Expected behaviour

All elements of the TE should be visible in the widget. 
## Suggestions

[Discusion in Slack.](https://toggl.slack.com/archives/C029WU7AY/p1586876828185600)
[Case.](https://app.intercom.com/a/apps/ayixs927/inbox/inbox/conversation/26635638832)

**Comment by kadrihvl** [comment]
You prob are aware already, but I'll add user's further comments, just in case:
>You made me thinking about the question yesterday about the battery saving options and dark mode, so I made some research on it, let me share my experience. I hope it will help your investigation.
As I have a Sony cell phone, the night/dark mode is not available by default. I have activated the Developer options and there is a Night mode setting there which has the options that can be seen in the attached image.
The battery saving option is named STAMINA here and has no effect on night mode.
So yesterday evening I saw that Toggl widget shows the title with whitish fonts on a black background which is fine, but this morning this was black on black again. So I went to the Developer options > Night mode and switched between the options and found the following: widget works well if Night mode is on but shows the title with black fonts on a black background if Night mode is off.
I hope this is an issue that can be fixed even if a newer Android version has been already released.
